src/components/backend-ai-credential-list.ts

```diff
@@ -696,25 +697,91 @@ export default class BackendAICredentialList extends BackendAIPage {
       `, root);
   }
 
+  _validateRateLimit() {
```

Comment:
I think we should use "[_addInputValidator](https://github.com/lablup/backend.ai-webui/blob/3ee743375ebe4fc26baa0a26d71a67394efbae7f/src/components/backend-ai-page.ts#L77-L126)" instead of adding custom validator in every components. Anyway let's handle them as a separate issue.

Reply:
I agree. It's better to be handled by https://github.com/lablup/backend.ai-webui/issues/1380.